.gitpod.yml

```diff
@@ -0,0 +1,12 @@
+image: gitpod/workspace-dotnet
+
+# List the ports you want to expose and what to do when they are served. See https://www.gitpod.io/docs/43_config_ports/
+ports:
+  - port: 5000
+    onOpen: ignore
+  - port: 5001
+    onOpen: open-preview
+
+# List the start up tasks. You can start them in parallel in multiple terminals. See https://www.gitpod.io/docs/44_config_start_tasks/
+tasks:
+  - command: cd WeatherForecast/ && dotnet run --urls http://0.0.0.0:5001
```

Comment:
The `part_2.md` instructions explicitly mentions going to https://localhost:5001/swagger/index.html which would fail if someone is using the browser app without adding "swagger/index.html" (if this workspace is opened in VSCode Desktop app then it should work fine). 

Learners should be able to figure this out but I'm pretty sure people will get confused by this.

Reply:
You're right, annoyingly we can't specify the right url of course, but we should flag something!